test/python/unittest/API/APITest.py

```diff
@@ -413,6 +413,158 @@ def testConstantRecords(self):
             if api.variants[b] is True and b in backend_filesupport:
                 self.makeConstantRoundTrip(backend_filesupport[b])
 
+    def makeEmptyRoundTrip(self, file_ending):
+        # write
+        series = api.Series(
+            "unittest_py_empty_API." + file_ending,
+            api.Access_Type.create
+        )
+
+        ms = series.iterations[0].meshes
+        SCALAR = api.Mesh_Record_Component.SCALAR
+        DT = api.Datatype
+
+        ms["CHAR"][SCALAR].make_empty(DT.CHAR, 1)
```

Comment:
if you do this (as well) with `np.dtype`'s as above this would be ideal

Reply:
The method `RecordComponent.make_empty` currently expects a `openpmd_api.Datatype`. Should I add an overload for numpy datatypes so the following is legal?
```python
series.iterations[0].meshes["E"]["x"].make_empty(numpy.dtype("int16"), 5)
```

Reply:
yep, such an additional overload would be good for usability.